support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -137,12 +139,14 @@ function isNumeric(str: string) {
 }
 
 const searchParams = new URLSearchParams(window.location.search);
-const queryAmount = searchParams.get('amount');
+const searchParamsPrice = searchParams.get('price');
```

Comment:
@paul-daniel-dempsey [I've changed this to `price`](https://github.com/guardian/support-frontend/pull/5955/commits/338254fbbcbe1771a683176dbd7dfe613ad7422a) to align with the product catalog.

Reply:
Yep aligning names to `price` is good, thanks.